mne_bids/tests/test_utils.py

```diff
@@ -49,12 +49,12 @@ def test_handle_datatype():
     n_channels = 2
     sampling_rate = 100
     data = random((n_channels, sampling_rate))
-    # datatype is given, check once for each datatype
-    channel_types = ['grad', 'eeg', 'ecog', 'seeg']
-    datatypes = ['meg', 'eeg', 'ieeg', 'ieeg']
-    for ch_type, datatype in zip(channel_types, datatypes):
+    channel_types = ['grad', 'eeg', 'ecog', 'seeg', 'dbs']
+    expected_modalities = ['meg', 'eeg', 'ieeg', 'ieeg', 'ieeg']
+    # do it once for each type ... and once for "no type"
+    for chtype, datatype in zip(channel_types, expected_modalities):
```

Comment:
```suggestion
    for ch_type, datatype in zip(channel_types, expected_modalities):
```
can you avoid such cosmetic changes? ch_type comes from mne-python variable naming convention.

thx

Reply:
Sorry, I think this change actually came from a rebase gone wrong, definitely wasn't intentional!

Reply:
Also added a whats_new entry now